tutorials/CRPS_for_Ensembles.ipynb

```diff
@@ -25,9 +25,11 @@
    "id": "595cd38f",
    "metadata": {},
    "source": [
-    "In weather forecasting it is common to run an ensemble of deterministic models. Each ensemble member provides a deterministic forecast. The idea is that the combined ensemble forecasts can be used to estimate a pdf. \n",
+    "In weather forecasting it is common to run an ensemble of deterministic models. Each ensemble member provides a deterministic forecast. The idea is that the combined ensemble forecasts can be used to estimate a Probability Density Function (PDF), and hence a CDF. \n",
```

Comment:
```suggestion
    "In weather forecasting it is common to run an ensemble of deterministic models. Each ensemble member provides a deterministic forecast. The idea is that the combined ensemble forecasts can be used to estimate a probability density function (PDF), and hence a CDF. \n",
```

Reply:
Unsure if this was meant to be capitalised or not. If it is meant to be capitalised in this context, please ignore my comment.